docs/logging/01-01-logging.md

```diff
@@ -3,10 +3,10 @@ title: Overview
 ---
 
 Logging in Kyma uses [Loki](https://github.com/grafana/loki) which is a Prometheus-like log management system. This lightweight solution, integrated with Grafana, is easy to understand and operate. The main elements of the logging stack include:
-* The Agent acting as a log router for Docker containers. It runs inside Docker, checks each container, and routes the logs to the log management system. Currently, Kyma supports the [Promtail](https://github.com/grafana/loki/tree/master/docs/clients/promtail) log collector. For details on log collector configuration, see [this](/components/logging/#tutorials-configure-the-log-collector) tutorial.
-* Loki main server which stores logs and processes queries.
-* [Grafana](https://grafana.com/) logging and metrics platform used for quering and displaying logs. 
 
+* The Agent acting as a log router for Docker containers. It runs inside Docker, checks each container, and routes the logs to the log management system. Currently, Kyma supports the [Fluent Bit](https://fluentbit.io/) log collector. For details on log collector configuration, see [this](/components/logging/#tutorials-configure-the-log-collector) tutorial.
```

Comment:
```suggestion
* The Agent acting as a log router for Docker containers. It runs inside Docker, checks each container, and routes the logs to the log management system. Currently, Kyma supports the [Fluent Bit](https://fluentbit.io/) log collector.
```
I removed the reference to the tutorial because it no longer exists (it was done for Fluent Bit when we still had Promtail).